mitodl/odl-video-service
# Updating JWT access token for EdxEndpoint

#### Pre-Flight checklist


#### What are the relevant tickets?
Fix #927

#### What's this PR do?
Sets expiration date on an existing access_token. Default is 0 so that should trigger a refresh when the EdxEndpoint instance is used. 

If the access token is expired it sends a request to the corresponding edx endpoint, then stores the new access_token and expires_in for that instance.

#### How should this be manually tested?
Set the values for `EDXORG_CLIENT_ID` and `EDXORG_CLIENT_SECRET` in the env file. Run migrations. 
Try uploading a video, when the video is done processing it should appear in your edx instance.
